src/types/Filters.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filters {
```

Comment:
```suggestion
export enum Filter {
```
use singular form for enum (it's a widespread way to name)